crates/rc-box/src/lib.rs

```diff
@@ -634,6 +635,50 @@ then the data will be pinned in memory and unable to be moved."),
 
         #[cfg(feature = "std")]
         impl<T: ?Sized> UnwindSafe for $RcBox<T> where Box<T>: UnwindSafe {}
+
+        #[cfg(feature = "unsize")]
+        doc_comment! {
+            concat!("Unsizes a pointer using the `unsize` crate.
+# Usage
```

Comment:
nitpick: there should be another newline before the header here to match the style of the rest of the documentation.